### DISCUSSION TITLE
Regarding fork, repositories etc.

### DISCUSSION THREAD

**Discussion by anmolsonthalia** [community/community#33604] [Other Feature Feedback, Questions, & Ideas]
How to hide or delete the fork, repositories from the account that I don't want to show to other user.

**Comment by radar-blip** [comment]
Open the repository you want to delete. Click on settings at the top of that repo. 
At the bottom of that page you will see a button to delete repository.

There is also a button there marked 'Change Visibility'. This will hide your repository from public view.

**Comment by SaikiranVoladri** [comment]
You can "hide" the repository from your GitHub profile by deleting the repository from GitHub, but leaving the local repository on your computer (or somewhere else). You can then later re-add that local repo to GitHub if needed.

**Comment by jfullstackdev** [comment]
if you still want to make your project available to anyone you want to share it with, then make it private, just add those collaborators, that way, you still have the project available to you and it's safe being on GitHub, but you still have the ability to share it with others and cannot be seen by any other user

**Reply by SaikiranVoladri** [reply]
hey bro i had changed the code as you said once check and merge it if there are any conflicts just message me

**Comment by SaraEOlivera** [comment]
I think you should make it private so that the project cannot be seen for others but you can have it available for you. In case you want to share it with others you can change the visibility again and make it public.
